### PR TITLE
update installation instructioins

### PR DESCRIPTION
**Pull request by MikePendo**

### CODE REVIEW

File: pnddocs/flutter.md [hunk 1]
```diff
@@ -20,6 +37,11 @@ In the `AppDelegate.m` add the follwing:
     return YES;
 }
 ```
+
+In order to enable Pendo pairing mode (tagging and testing) select your project select the info tab and add Url Type with pendo url scheme
```
Comment (noambartouv) [inline pnddocs/flutter.md:41]: To enable Pendo pairing mode (tagging and test on device) go to your application subscription, select the info tab and create a URL Type using the Pendo url scheme

Comment (noambartouv) [reply pnddocs/flutter.md:41]: change in all places

Comment (MikePendo) [reply pnddocs/flutter.md:41]: _go to your application subscription_ he need to go to the project In Xcode .... why application subscription?

Comment (MikePendo) [reply pnddocs/flutter.md:41]: maybe:
In order to enable Pendo pairing mode (tagging and testing) select your project **_then_** select the info tab and add Url Type with pendo url scheme

Comment (noambartouv) [reply pnddocs/flutter.md:41]: Sorry thought this was something else. So let's go with:

To setup the Pendo pairing mode (tagging and test on device) select your project, navigate to the relevant target, select the info tab and create a URL Type using the Pendo url scheme (found in your subscription under the App Details tab)

**Pull request merged**